package.json

```diff
@@ -34,6 +34,6 @@
     "xml": "^1.0.0"
   },
   "peerDependencies": {
-    "mocha": "^2.2.5"
+    "mocha": "^3 || ^2.2.5"
```

Comment:
@vdh @michaelleeallen Actually, reading the semver spec I like the versioning as you have it specified. We'll probably only care to support the most recent version of the `2.x` line and the `3.x` line to keep things simple